tests/fixture/mod.rs

```diff
@@ -0,0 +1,126 @@
+use git2;
+use git_rsl::{self, BranchName, RemoteName};
+use tempfile::{self, TempDir};
+use assert_cli::{Assert, Environment};
+
+#[allow(dead_code)]
+pub struct TestFixture {
+    origin: TempDir,
+    local: TempDir,
+    gpg_home: TempDir,
+    pub env: Environment
+}
+
+use std::fs::File;
+use std::io::Write;
+
+fn write_gpg_keygen_script(out_path: &str) {
+    let mut f = File::create(out_path).unwrap();
+    f.write_all("%no-protection\n\
+                 %transient-key\n\
+                 Key-Type: DSA\n\
+                 Key-Length: 1024\n\
+                 Name-Email: test_user@polysync.io\n\
+                 Expire-Date: 0\n\
+                 %commit\n".as_bytes()
+    ).unwrap();
+}
+
+use std::process::Command;
+fn first_gpg_key_id(gpghome: &str) -> String {
+    let out = Command::new("gpg2")
+        .env("GNUPGHOME", gpghome)
+        .args(vec!["--with-colons", "--list-keys"])
+        .output()
+        .unwrap();
+
+//    "pub:u:1024:17:31BC51B4BB930CA9:1528761082:::u:::scaSCA:::::::"
```

Comment:
Can this be removed?

Reply:
I left it in as an example of what the data looks like, so  the following parsing step makes sense. I'll amend this with more comments about what it means.